src/components/ha-date-range-picker.ts

```diff
@@ -249,6 +251,12 @@ export class HaDateRangePicker extends LitElement {
         <div slot="input" class="date-range-inputs" @click=${this._handleClick}>
           ${!this.minimal
             ? html`<ha-svg-icon .path=${mdiCalendar}></ha-svg-icon>
+                <ha-icon-button-prev
+                  .label=${this.hass.localize("ui.common.previous")}
+                  class="prev"
+                  @click=${this._handlePrev}
+                >
+                </ha-icon-button-prev>
```

Comment:
Please hide prev and next buttons on mobile, it takes too much space, look how it is done with `ha-svg-icon`

Reply:
Maybe a stupid question, why is there a second Date-Entry for EndDate? Both: StartDate and Enddate load the exact same Daterange-Picker which includes Start and Enddate? Thought about to remove the EndDate for mobile. 
If i disable the prev and next buttons they can not be used on small displays. Maybe I found another solution: changing the media-query for the mentioned ha-svg-icon {display: none} solved it.

Reply:
It is there to see the time range, but sure we could show the time range in a more compact way then an input field. If you want to try it out, please open a seperate PR.
For now please hide prev and next buttons on mobile and in the next steps we can improve it, maybe with help of UX.

Reply:
ok, thank you